staking/contracts/contracts.py

```diff
@@ -241,6 +262,12 @@ def init(
     """Initialise the newly deployed contract, funding it with a minimum
```

Comment:
Even though pyteal provides sensible `OnComplete` and _on creation_ defaults, as an auditor I would be prefer more explicitness here (also in keeping with item 2 of the [Zen of python](https://peps.python.org/pep-0020/#the-zen-of-python))

Reply:
So consider either specifying the actions allowed, or adding a comment which explains which actions are being allowed due to PyTeal's conventions.